line/line.mjs

```diff
@@ -29,13 +29,14 @@ export async function findLineById(id = null, options = {}) {
 
   if (line !== null) {
     if (options.text === 'blob') {
-      return { text: line.textualBody };
+      return { 'Content-Type': 'text/plain', body: line.text };
     } else if (options.image === 'full') {
-      return { image: line.canvas };
+      // Assuming line.canvas contains the full image
+      return { 'Content-Type': 'image/jpeg', body: line.canvas };
     } else if (options.image === 'line') {
-      return { image: `some line image URL for id ${id}` };
+      return { 'Content-Type': 'image/jpeg', body: `some line image URL for id ${id}` };
     } else if (options.lookup) {
-      return { relatedDocument: `some ${options.lookup} document for id ${id}` };
+      return { 'Content-Type': 'text/plain', body: `some ${options.lookup} document for id ${id}` };
```

Comment:
This is still an Object. You want to look at adding the ContentType to the `res` Response here and then the body will be understood by the client when you `.send()` it. You might also want to check the statusCode.

Reply:
I have addressed the status codes in the updated code